sass/components/_emojisprite.scss

```diff
@@ -7,6 +7,7 @@
     height: 66px;
     background-repeat: no-repeat;
     cursor: pointer;
+    -moz-transform: scale(0.35);
     transform-origin: 0 0;
     // Using zoom for now as it results in less blurry emojis on Chrome - MM-34178
     zoom: 0.5;
```

Comment:
Should this be `-moz-transform: scale(0.5);` since `zoom: 0.5;`?

Reply:
this was the code before i deleted it, thinking it was obsolete